assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -182,7 +186,8 @@ const onCreateOneOffPayPalPaymentResponse =
 
         // For PayPal create payment errors, the Payment API passes through the
```

Comment:
Converting PayPal API errors > CheckoutFailureReasons in payment-api is still on my to-do list!